user_pinger.py

```diff
@@ -208,16 +220,26 @@ def handle_comment(self, comment: praw.models.Comment) -> None:
         except ValueError:
             # no trigger
             return
-        else:
-            self.logger.debug("Ping found in %s", str(comment))
-            try:
-                trigger: str = split[index + 1]
-            except IndexError:
-                self.logger.debug("End of comment with no group specified")
-                return
-            else:
-                self.logger.debug("Found group is %s", trigger)
-                self.handle_ping(trigger, comment)
+        self.logger.debug("Ping found in %s", str(comment))
+        try:
+            token: str = split[index + 1]
+        except IndexError:
+            self.logger.debug("End of comment with no group specified")
+            return
+        self.logger.debug("Comment contained group token %s", token)
+
+        # Strip punctuation like:
+        # !ping WEEBS, what do you folks think of this?
+        trigger: str = token.strip(self.GROUP_STRIP_PUNCT)
+
+        # Validate group name to save having to look up invalid names
+        ret: bool, msg: Optional[str] = self._validate_group_name(trigger)
```

Comment:
I get a syntax error on this line using Python 3.6.9

Based on [PEP 526](https://www.python.org/dev/peps/pep-0526/) and [this stackoverflow thread](https://stackoverflow.com/questions/52082939/type-hints-when-unpacking-a-tuple) I don't think unpacking a tuple like this is allowed in any version of python.

```suggestion
        ret: bool
        msg: Optional[str]
        ret, msg = self._validate_group_name(trigger)
```